frontend/public/components/overview/project-overview.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-unused-vars, no-undef */
```

Comment:
Do we want to disable these rules for the whole file? I feel like we should still be checking the rest of the file and only 'whitelisting' the specific instances where we know we broke these rules intentionally.

Reply:
TypeScript is already checking it for us, so these eslint rules are unnecessary for tsx files.

Reply:
Yeah, just verified:

```
ERROR in /Users/sam/go/src/github.com/openshift/console/frontend/public/components/overview/project-overview.tsx(25,5):
TS6133: 'foo' is declared but its value is never read.
ERROR in /Users/sam/go/src/github.com/openshift/console/frontend/public/components/overview/project-overview.tsx(26,1):
TS2304: Cannot find name 'bar'.
```

We have it set in our tsconfig:

https://github.com/openshift/console/blob/master/frontend/tsconfig.json#L14